Engine/Helper.cs

```diff
@@ -299,11 +305,12 @@ public PSModuleInfo GetModuleManifest(string filePath, out IEnumerable<ErrorReco
             Collection<PSObject> psObj = null;
             using (var ps = System.Management.Automation.PowerShell.Create())
             {
+                ps.RunspacePool = _runSpacePool;
+                ps.AddCommand("Test-ModuleManifest");
```

Comment:
Just a suggestion, but the builder patter fluent syntax often feels more explicit about context to me:

```csharp
ps.AddCommand("Test-ModuleManifest")
    .AddParameter("Path", filePath)
    .AddParameter("WarningAction", ActionPreference.SilentlyContinue)
```

Also, is `SilentlyContinue` supported in PSv3?

Reply:
Yes, I can definitely use the build pattern here, the only cases where I don't like it is when there are actions where I want to debug step by step.
According to [this](https://devblogs.microsoft.com/powershell/erroraction-and-errorvariable/) blog post from 2006 (PS v1), `SilentlyContinue` was already available.

Reply:
Quite right -- compatibility catalogs confirm

Reply:
(I say this because mysteriously some official blog posts, with changelogs in them, are wrong)